content/id/contribute/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+content_template: templates/concept
+title: Berkontribusi ke dokumentasi Kubernetes
```

Comment:
```suggestion
title: Berkontribusi ke Dokumentasi Kubernetes
```

Reply:
Addressed here https://github.com/kubernetes/website/pull/18452/commits/8237a3870e695d3dd41d3f2411b5898008b21c5d

I also added an update to a mistranslation to Open issue -> I thought this was a noun, but it's actually a verb.
Thanks @wahyuoi for pointing this out!
https://github.com/kubernetes/website/pull/18452/commits/8237a3870e695d3dd41d3f2411b5898008b21c5d#diff-ffc999e20875f5664de07a3c269c7118R51